src/labeler.ts

```diff
@@ -24,7 +28,7 @@ export async function run() {
       return;
     }
 
-    const client: ClientType = github.getOctokit(token);
+    const client: ClientType = github.getOctokit(token, {}, pluginRetry.retry);
```

Comment:
So I've noticed that when using the [setLabels API](https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#set-labels-for-an-issue), it would still fail with 5xx error if adding >50 labels (or if there's currently >50 labels and we're trying to remove them all), but retrying the query then passes.

Thus I'm simply adding the `retry` plugin instead. I think it's a good thing to add even if that issue gets officially fixed by Github on the backend, for resilience.